src/components/GlobalHeader/index.js

```diff
@@ -69,11 +69,10 @@ class GlobalHeader extends Component {
   };
 
   logoutSession = () => {
-    const { dispatch } = this.props;
+    const { dispatch, auth } = this.props;
```

Comment:
The `auth` namespace is declared but its value is never read. Can we consider removing this reference?